src/DynamoSandbox/Program.cs

```diff
@@ -102,5 +112,30 @@ private static void UpdateSystemPathForProcess()
                         EnvironmentVariableTarget.Process) + ";" + DynamoCorePath;
             Environment.SetEnvironmentVariable("Path", path, EnvironmentVariableTarget.Process);
         }
+        
+        /// <summary>
+        /// If Dynamo Sandbox fails to acquire Dynamo Core path, show a dialog that
+        /// redirects to download DynamoCore.msi, and the program should exit.
+        /// </summary>
+        private static void NotifyUserDynamoCoreUnresolved()
```

Comment:
DynamoSandbox has its resources files inside the Dynamo Core folder; I'm afraid if the `Detective.dll` would not be able to resolve the resource files without resolving the Dynamo Core first (it must have failed to resolve Dynamo Core in order to reach this point of code).

@sharadkjaiswal Do you have any other possible solution for this?